src/themes/teams/components/Button/buttonVariables.ts

```diff
@@ -6,6 +6,7 @@ export interface IButtonVariables {
   height: string
   minWidth: string
   maxWidth: string
+  defaultBorderRadius: string
```

Comment:
are we using any different other than `default` one for `Button`? If not, we should probably consider to rename it to something like just `borderRadius` - what do you think?

Reply:
Yes, we have circularRadius as well, that is why this is named like that.